### ISSUE TITLE
`\setmonofont` broken

### ISSUE THREAD

**Issue by Stone-Zeng** [wspr/fontspec#300]
The following codes will give an error:
```tex
\documentclass{article}
\usepackage{fontspec}
\setmonofont{TeX Gyre Cursor}

\begin{document}
  Hello, world!
  \texttt{Hello, world!}
  \textsf{Hello, world!}
\end{document}
```

In the log file, I can find this:
```tex
! Undefined control sequence.
<argument> \l__fontspec_addfontfeatures_bool 
                                             
l.3 \setmonofont{TeX Gyre Cursor}
                                 
? 
```

XeLaTeX and LuaLaTeX will give the same result. However, when use `\setsansfont` or `\setmainfont`, it wil be OK.

I use `fontspec` v2.6b on TeX Live 2017.

**Comment by Stone-Zeng** [comment]
This bug seems to be fixed in v2.6c.

**Comment by wspr** [comment]
Thanks for taking the time to report the bug and apologies for the inconvenience.
